addons/udes_warehouse_classification/models/product_template.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    u_product_warehouse_classification_ids = fields.Many2many(
+        comodel_name="product.warehouse.classification",
+        string="product_warehouse_classification_ids",
+        help="Classifications on products used for messaging purposes.",
+    )
+
+    def get_classification_messages_for_label(self, label_name):
```

Comment:
shall we call it report id or report ref? we haven't really modelled "labels"

Reply:
Sure, no problem